### DISCUSSION TITLE
Add the combo-burst option

### DISCUSSION THREAD

**Discussion by greenmg2003** [ppy/osu#16578] [Ideas] [answered]
I have an idea to make osu! lazer better. There are some cool skins that has the combo burst images. In that way, there should be the combo-burst option to let the player choose whether to turn on or off there.
Ah, and the system of combo burst, just exact 50× combo and 1 image of combo-burst will appear. If combo get 1000+, 100 combo/1 combo-burst image.
Thank you for reading, and hope I can get the answer soon!

**Comment by frenzibyte** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/4629

**Reply by greenmg2003** [reply]
Thanks for informing! I'll check it out!
